packages/nextjs-mf/src/plugins/container/InvertedContainerPlugin.ts

```diff
@@ -18,6 +24,7 @@ interface InvertedContainerOptions extends ModuleFederationPluginOptions {
```

Comment:
The 'InvertedContainerOptions' interface is not used in this diff. It would be better to use this interface to type check the options passed to the 'InvertedContainerPlugin' class.